cmd/apply.go

```diff
@@ -68,6 +68,7 @@ func NewApplyCmd(globalCfg *config.GlobalImpl) *cobra.Command {
 	f.StringVar(&applyOptions.PostRenderer, "post-renderer", "", `pass --post-renderer to "helm template" or "helm upgrade --install"`)
 	f.StringArrayVar(&applyOptions.PostRendererArgs, "post-renderer-args", nil, `pass --post-renderer-args to "helm template" or "helm upgrade --install"`)
 	f.StringVar(&applyOptions.Cascade, "cascade", "", "pass cascade to helm exec, default: background")
+	f.StringVar(&applyOptions.DryRun, "dry-run", "", "pass dry-run to helm exec")
```

Comment:
I checked the documentation of helm and found that dry-run is a bool switch, not a string type parameter. Please correct me if I understand it wrong.
for example:
```
helm install --debug --dry-run goodly-guppy ./mychart
```
[doc](https://helm.sh/docs/chart_template_guide/getting_started/#adding-a-simple-template-call)

Reply:
That was my fault.  I assumed it was the same as June to apply, but seems like it only does a server mode and not a client mode.  I’ll make the changes.